pyplanet/utils/log.py

```diff
@@ -30,7 +30,7 @@ def get_client(cls):
 		if not cls.CLIENT:
 			cls.CLIENT = Client(
 				dsn='https://bcbafd2d81234d53b32c66077ae0a008:98dcc46acc484eeb95ebf9f0c30e6dd4@sentry.s3.toffe.me/2',
```

Comment:
Is Sentry running again? Otherwise we might as well remove it.

Reply:
It's not for now, do you think it's worth it to start it again? I will upgrade it, clean it and start it if you think so.